### PR TITLE
Update task & cache BBEs

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import ballerina/io;
+import ballerina/task;
+import ballerina/time;
+
+// Creating a job to execute by the scheduler
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:5]:
```suggestion
// Creating a job to be executed by the scheduler.
```

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+class Job {
+
+    *Job;
+    int i = 1;
+
+    // Executes by the Scheduler when the scheduled trigger fires
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:11]:
```suggestion
    // Get executed by the scheduler when the scheduled trigger fires.
```

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    isolated function init(int i) {
+        self.i = i;
+    }
+}
+
+public function main() {
+
+    // Get the current time
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:24]:
```suggestion
    // Get the current time.
```

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    time:Utc currentUtc = time:utcNow();
+    // Increase time by three second
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:26]:
```suggestion
    // Increase the time by three seconds.
```

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    time:Utc newTime = time:utcAddSeconds(currentUtc, 3);
+    // Get `time:Civil` for the given time
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:28]:
```suggestion
    // Get the `time:Civil` for the given time.
```

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    time:Civil time = time:utcToCivil(newTime);
+    // Creates a `time:ZoneOffset` using the given configuration.
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:30]:
```suggestion
    // Create a `time:ZoneOffset` using the given configuration.
```

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    time:ZoneOffset zoneOffset = {hours: 5, minutes: 30};
+    // Set the offset into `time:Civil`
+    time.utcOffset = zoneOffset;
+
+    // Schedule the frequency job
```
Comment (praneesha) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:35]:
```suggestion
    // Schedule the frequency job.
```

File: examples/task-frequency-job-execution/task_frequency_job_execution.out [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# The `execute()` function is triggered every second, navigate to the directory that contains the
```
[the remark this answers is not in the document]
Comment (kalaiyarasiganeshalingam) [reply examples/task-frequency-job-execution/task_frequency_job_execution.out]: Removed.

**Review by praneesha — APPROVED** [review]

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    // Get the current time.
+    time:Utc currentUtc = time:utcNow();
+    // Increase the time by three seconds.
+    time:Utc newTime = time:utcAddSeconds(currentUtc, 3);
+    // Get the `time:Civil` for the given time.
+    time:Civil time = time:utcToCivil(newTime);
+    // Create a `time:ZoneOffset` using the given configuration.
+    time:ZoneOffset zoneOffset = {hours: 0, minutes: 0};
+    // Set the offset to the `time:Civil`.
+    time.utcOffset = zoneOffset;
```
Comment (daneshk) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:33]: If the zone offset is 0. we don't need to add it. shall we remove it

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+    isolated function init(int i) {
+        self.i = i;
+    }
+}
+
+public function main() returns error? {
+
+    // Get the current time.
+    time:Utc currentUtc = time:utcNow();
+    // Increase the time by three seconds.
```
Comment (daneshk) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:26]: From a task module perspective, this means we are delaying the task by 03 seconds. shall we add it in the comment

File: examples/task-frequency-job-execution/task_frequency_job_execution.bal [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+    time:Utc newTime = time:utcAddSeconds(currentUtc, 3);
+    // Get the `time:Civil` for the given time.
+    time:Civil time = time:utcToCivil(newTime);
+    // Create a `time:ZoneOffset` using the given configuration.
+    time:ZoneOffset zoneOffset = {hours: 0, minutes: 0};
+    // Set the offset to the `time:Civil`.
+    time.utcOffset = zoneOffset;
+
+    // Schedule the frequency job.
```
Comment (daneshk) [inline examples/task-frequency-job-execution/task_frequency_job_execution.bal:31]: mention we are scheduling the job interval as 1 second

**Review by daneshk — APPROVED** [review]

**Pull request merged**